app/qml/layers/MMFeaturesListPage.qml

```diff
@@ -21,6 +21,7 @@ MMComponents.MMPage {
 
   property var selectedLayer: null
   property bool hasToolbar: false
+  property bool layerIsReadOnly: featuresModel.layer?.readOnly ?? false
```

Comment:
It seems like we could go just with this
```suggestion
  property bool layerIsReadOnly: selectedLayer?.readOnly ?? false
```
that way we do not add dependency for `featuresModel`